website/docs/src/content/cloud/index.mdx

```diff
@@ -0,0 +1,356 @@
+const result = await MyWorkflow.run(
+  { input: "User query" },
+  printUrl: true // log the tracing URL
+);
+```
+
+The trace viewer provides multiple ways to analyze workflow execution:
+
+- **Timeline view**: See how long each component took and their sequence of execution
+- **Component tree**: Navigate the hierarchical structure of your workflow
+- **Input/output inspector**: Examine the exact data flowing between components
+- **Error highlighting**: Quickly identify where failures occurred
+
+![Component trace](/cloud/component-trace.png)
+
+For more details see the full [observability reference](/docs/cloud/observability).
+
+## Local development
+
+GenSX provides a seamless development experience that mirrors the cloud environment, allowing you to build and test your workflows locally before deployment:
+
+### Development Server
+
+The `gensx start` command launches a local development server that:
+
+- Compiles your TypeScript workflows on the fly
+- Automatically generates schemas for your workflows
+- Creates local REST API endpoints identical to the cloud endpoints
+- Hot-reloads your code when files change
+- Provides the same API shape locally as in production
+
+```bash
+# Start the development server with a TypeScript file
+npx gensx start ./src/workflows.tsx
+
+🔍 Starting GenSX Dev Server...
+ℹ Using project name from gensx.yaml: my-app
+ℹ Starting development server...
+✔ Compilation completed
+✔ Generating schema
+
+📋 Available workflows:
+- MyWorkflow: http://localhost:1337/local/my-app/default/workflows/MyWorkflow/run
+
+✅ Server is running. Press Ctrl+C to stop.
+```
+
+### Local storage providers
+
+GenSX provides local implementations for most storage providers, enabling development without cloud dependencies:
+
+- **BlobProvider**: Uses local filesystem storage (`.gensx/blobs`) for development
+- **DatabaseProvider**: Uses local SQLite databases (`.gensx/databases`) for development
+- **SearchProvider**: Connects to the cloud vector search service even in development mode
+
+The local APIs mirror the cloud APIs exactly, so code that works locally will work identically when deployed:
+
+```tsx
+// This component works the same locally and in the cloud
+const SaveData = gensx.Component<{ key: string; data: any }, null>(
+  "SaveData",
+  async ({ key, data }) => {
+    // Blob storage works the same locally (filesystem) and in cloud
+    const blob = useBlob(`data/${key}.json`);
+    await blob.putJSON(data);
+    return null;
+  },
+);
+```
+
+For more details see the full [local development reference](/docs/cloud/local-development).
+
+## Projects & environments
+
+GenSX Cloud organizes your workflows and deployments using a flexible structure of projects and environments:
+
+- **Projects**: A collection of related workflows that are deployed together, typically corresponding to a codebase or application. Projects help you organize and manage your AI components as cohesive units.
```

Comment:
The current formatting makes this hard to read because the following paragraphs don't get nested under the bullet
```suggestion
**Projects** are a collection of related workflows that are deployed together, typically corresponding to a codebase or application. Projects help you organize and manage your AI components as cohesive units.
```